### ISSUE TITLE
If a prompt spans multiple lines, keystrokes result in the prompt label being displayed multiple times

### ISSUE THREAD

**Issue by bcongdon** [manifoldco/promptui#143]
**Bug description:** If the label of a prompt exceeds the horizontal width of the terminal window, then each keystroke results in the prompt being displayed on a newline.

**Expected behavior:** If a label spans multiple lines, the behavior is identical to if the label spans one line -- the label is displayed only once.

**Observed behavior:** When a label spans multiple lines, each keystroke results in a new printed version of the label, i.e. https://asciinema.org/a/hHDOBInyBFwBPXN2ZfaPcULKP

---

Minimal working example:

```go
package main

import (
        "strings"

        "github.com/manifoldco/promptui"
)

func main() {
        prompt := promptui.Prompt{
                Label:     strings.Repeat("Foo", 100),
        }
        prompt.Run()
}
```

Asciinema recording of the above: https://asciinema.org/a/hHDOBInyBFwBPXN2ZfaPcULKP

**Comment by bcongdon** [comment]
Duplicate of #122
